2-mandatory/5-eligible-students.js

```diff
@@ -7,7 +7,14 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
 
-function getEligibleStudents() {}
+function getEligibleStudents(array) {
+  let eligible = array.filter((student) => student[1] >= 8);
+  let eligibleStudents = [];
+  for (let student of eligible) {
+    eligibleStudents.push(student[0]);
+  }
+  return eligibleStudents;
+}
```

Comment:
this is good, I really like how you're using your variable names to provide context, it makes your solutions really readable. Instead of a for loop, another approach could be to chain a .map onto your filter to get the first item in each array